dali/operators/image/crop/bbox_crop.cc

```diff
@@ -96,191 +96,232 @@ struct Range {
+  If you do not provide a ``total_num_attempts`` value, this can result in an infinite
+  loop if the provided arguments cannot be satisfied.
+)code",
         -1)
     .AddOptionalArg(
         "all_boxes_above_threshold",
-         R"code(If true, all bounding boxes in a sample should overlap with the cropping window as specified by
-``thresholds``, otherwise the cropping window is considered invalid. If false, a cropping window will be considered 
-valid if any bounding box overlaps it sufficiently.)code",
+         R"code(If set to ``True``, all bounding boxes in a sample should overlap with the cropping
+window as specified by ``thresholds``.
+
+If the bounding boxes do not overlap, the cropping window is considered to be invalid. If set to
+``False``, and a bounding box overlaps the window, the window is considered to be valid.)code",
          true)
     .AddOptionalArg(
         "allow_no_crop",
-        R"code(If true, not cropping will be one of the possible outcomes of the random process, as
-if it was one more ``thresholds`` value to choose from.)code",
+        R"code(If set to ``True``, one of the possible outcomes of the random process will
+be to not crop, as if the outcome was one more ``thresholds`` value from which to choose.)code",
         true)
     .AddOptionalArg<int>(
         "crop_shape",
         R"code(If provided, the random crop window dimensions will be fixed to this shape.
 
 The order of dimensions is determined by the layout provided in ``shape_layout``.
 
-Note: ``crop_shape`` and ``input_shape`` should be provided together, and providing those is
-incompatible with using ``scaling`` and ``aspect_ratio`` arguments.)code",
+.. note::
+  ``crop_shape`` and ``input_shape`` should be provided together and providing those
+  arguments is incompatible with using ``scaling`` and ``aspect_ratio`` arguments.)code",
         std::vector<int>{}, true)
     .AddOptionalArg<int>(
         "input_shape",
         R"code(Specifies the shape of the original input image.
 
-The order of dimensions is determined by the layout provided in ``shape_layout``.
+The order of dimensions is determined by the layout that is provided in ``shape_layout``.
 
-Note: ``crop_shape`` and ``input_shape`` should be provided together, and providing those is
-incompatible with using ``scaling`` and ``aspect_ratio`` arguments.)code",
+.. note::
+  ``crop_shape`` and ``input_shape`` should be provided together but providing those arguments
+  is incompatible when using ``scaling`` and ``aspect_ratio`` arguments.
+)code",
         std::vector<int>{}, true)
     .AddOptionalArg<TensorLayout>(
         "bbox_layout",
         R"code(Determines the meaning of the coordinates of the bounding boxes.
-  Possible values are:
-
-  ``x`` (horizontal start anchor), ``y`` (vertical start anchor), ``z`` (depthwise start anchor),
 
-  ``X`` (horizontal end anchor),   ``Y`` (vertical end anchor),   ``Z`` (depthwise end anchor),
+Here are the possible values:
```

Comment:
The value of this argument is a string containing the following characters:

Reply:
Done